application/backend/app/supported_models/hyperparameters/base_model_no_extra.py

```diff
@@ -1,11 +0,0 @@
-# Copyright (C) 2025 Intel Corporation
-# SPDX-License-Identifier: Apache-2.0
-
-
-from pydantic import BaseModel, ConfigDict
-
-
-class BaseModelNoExtra(BaseModel):
-    """Base model class that does not allow for extra fields."""
-
-    model_config = ConfigDict(extra="forbid")
```

Comment:
If I'm not wrong, Geti uses `BaseModelNoExtra` to validate that the configuration elements do not contain unrecognized keys. For example, if the UI/client provides a parameter key with a typo, it's better to be strict and raise an error instead of silently ignoring that parameter.

Why are you removing this base model? Did you find a different solution to address the above problem?